tests/testquic.nim

```diff
@@ -24,13 +24,18 @@ proc createServerAcceptConn(
   .} =
     try:
       let conn = await server.accept()
+      if conn == nil:
+        return
+
       let stream = await getStream(QuicSession(conn), Direction.In)
       var resp: array[6, byte]
       await stream.readExactly(addr resp, 6)
       check string.fromBytes(resp) == "client"
 
       await stream.write("server")
       await stream.close()
+    except QuicTransportError:
+      discard # Transport failed (maybe dial error / handshake err, etc)
```

Comment:
i am not sure about this. if `QuicTransportError` is raised then it implies that `CatchableError` has happened, if it do happen then `discard` doesn't seem right because we don't expect that to happen so we need to brake the tests.

aslo, last time i added this `try` block i was lazy, we should only guard `let conn = await server.accept()` this line with try block.